alacritty_macos.yml

```diff
@@ -205,6 +205,10 @@ key_bindings:
 mouse_bindings:
   - { mouse: Middle, action: PasteSelection }
 
+multiclick_threshold: 300
```

Comment:
The units should be part of the name here to avoid ambiguity. Eg: `multiclick_threshold_ms`